ambry-server/src/main/java/com/github/ambry/server/AmbryRequests.java

```diff
@@ -693,6 +695,34 @@ public void handleReplicaMetadataRequest(NetworkRequest request) throws IOExcept
             logger.trace("{} Time used to find entry since: {}", partitionId,
                 (SystemTime.getInstance().milliseconds() - partitionStartTimeInMs));
 
+            // Compute CRC for backup recovery and verification
+            if (replicaMetadataRequestInfo.getReplicaPath().startsWith(BackupCheckerThread.DR_Verifier_Keyword)) {
+              Store storeToGet = storeManager.getStore(partitionId);
```

Comment:
you don't need a new variable `storeToGet` here, the `store` before this if statement can be used here.